network-api/networkapi/templates/pages/campaign.html

```diff
@@ -17,20 +17,32 @@
       {% else %}
       <div class="col-md-6">
       {% endif %}
-        {% with page.campaign.petition as petition %}<div
-          class="sign-petition full-width mb-5"
-          data-csrf-token="{{ csrf_token }}"
-          data-petition-id="{{ page.campaign.petition.id }}"
-          data-cta-header="{{ page.campaign.petition.header | escape }}"
-          data-cta-description="{{ page.campaign.petition.description | escape }}"
-          data-newsletter="{{ page.campaign.petition.newsletter }}"
-          data-checkbox1="{{ page.campaign.petition.checkbox_1 | escape }}"
-          data-checkbox2="{{ page.campaign.petition.checkbox_2 | escape }}"
-          data-share-link="{{ page.campaign.petition.share_link }}"
-          data-share-text="{{ page.campaign.petition.share_link_text }}"
-          data-thank-you="{{ page.campaign.petition.thank_you }}"
-        ></div>{% endwith %}
+        {% with page.campaign.petition as petition %}
+          {% if page.campaign.petition.share_twitter %}
```

Comment:
again, not completely related to you pr but everything from line 21 to 41, can you change `page.campaign.petition` to just `petition`

Reply:
Is that going to just work, or do I need to do a sort of
```
var petition = page.campaign.petition
```
type thing? Don't know off hand what the syntax for that would be so I used js to make it easier.